test/init/initialize.test.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable jest/prefer-strict-equal */
+const { promises: fs } = require('fs');
+const initUtils = require('../../dist/src/cmds/init/initutils');
+const readlineUtils = require('../../dist/src/utils/readline');
+const miscUtils = require('../../dist/src/utils/misc');
+const { initHandler } = require('../../dist/src/cmds/init/initialize');
+
+describe('initialize', () => {
+  describe('initHandler', () => {
+
+    afterEach(() => {
+      jest.clearAllMocks();
+      jest.restoreAllMocks();
+    });
+
+    const mockPackage = {
+      name: 'foo',
+      version: '1.0.0',
+      description: '',
+      main: 'index.js',
+      scripts: { test: 'echo "Error: no test specified" && exit 1' },
+      author: '',
+      license: 'ISC',
+    };
+
+    const mockArgv = { foo: 'bar' };
+    const mockWallet = [
+      {
+        bundle: {},
+        initialPermissions: {},
+      },
+      { dist: 'dist', outfileName: 'bundle.js', port: 8081 },
+    ];
+    const expectedReturnValue = {
+      foo: 'bar',
+      src: 'index.js',
+      dist: 'dist',
+      outfileName: 'bundle.js',
+      port: 8081,
+    };
+
+    it('function successfully executes under normal conditions', async () => {
+      jest.spyOn(console, 'log').mockImplementation();
+      const fsWriteMock = jest.spyOn(fs, 'writeFile').mockImplementation(() => true);
+      const asyncPackageInitMock = jest.spyOn(initUtils, 'asyncPackageInit').mockImplementation(() => mockPackage);
+      const validateEmptyDirMock = jest.spyOn(initUtils, 'validateEmptyDir').mockImplementation();
+      const buildWeb3WalletMock = jest.spyOn(initUtils, 'buildWeb3Wallet').mockImplementation(() => mockWallet);
+      const closePromptMock = jest.spyOn(readlineUtils, 'closePrompt').mockImplementation(() => mockPackage);
+      expect(await initHandler(mockArgv)).toStrictEqual(expectedReturnValue);
```

Comment:
We should add a newline here, and in the same spot for the other test cases, for readability (and navigability for those of us that use vim 😄 ).